src/cr/cube/cubepart.py

```diff
@@ -248,6 +249,29 @@ def __init__(self, cube, slice_idx, transforms, population, mask_size):
         self._population = population
         self._mask_size = mask_size
 
+    def __repr__(self):
```

Comment:
Can this property be generalized in the superclass `CubePartition` ? I think so

Reply:
It operates differently on both subclasses, and I believe the behavior is different enough that it warrants extracting the method into each subclass. For strand, e.g., you don't need to handle row labels, while for the slice you do.